src/procfs.c

```diff
@@ -292,5 +357,15 @@ main(int argc, char **argv)
 		return (test_fail(TST_NAME, "procfs 6 - writable files"));
 	}
 
+	/* Test 7 - /proc/sys/kernel/random/boot_id */
+	if (check_kernal_random_boot_id() != 0)  {
```

Comment:
Ooops, spelling mistake:  "kernel", not "kernal".  (Are you an old Amiga person?)

Reply:
No, just a combination of a typo and then dabbrev-expand I guess. (Atari, not Amiga). Will fix.